Scripts/GameCode/ODIN/Components/ScriptedDamageManagerComponent.c

```diff
@@ -0,0 +1,42 @@
+
+//------------------------------------------------------------------------------------------------
+modded class ScriptedDamageManagerComponent : BaseScriptedDamageManagerComponent
+{
+	[RplProp(onRplName: "ODIN_OnEnableDamageValueUpdated")]
```

Comment:
Seems excessive to add the `OnDamage` handle for all machines. Did you check whether it would be sufficient to just add it for the server?

Reply:
As in you mean to add a check in ``ODIN_OnEnableDamageValueUpdated()`` to only insert/remove the handler, if it's the server? So just exit the function if called from non-server?

Reply:
No, I would drop the RplProp all together and just do all the handling on the server

Reply:
Makes sense. Updated.